Modules/nismodule.c

```diff
@@ -44,12 +44,48 @@ PyDoc_STRVAR(maps__doc__,
 Returns an array of all available NIS maps within a domain. If domain\n\
 is not specified it defaults to the system default domain.\n");
 
-static PyObject *NisError;
+typedef struct {
+    PyObject *NisError;
```

Comment:
While "NisError" name makes sense for a global, maybe "nis_error" name for a structure member is better. I don't know.